### ISSUE TITLE
[TODO OSPO Ambassador] Request - Jose Rodriguez

### ISSUE THREAD

**Issue by dagoshe** [todogroup/governance#297]
# TODO OSPO Ambassador Request

This is an issue created to request to become an OSPO Ambassador. If you (as individual) are interested to apply, please make sure you have (1) read the [TODO OSPO Ambassador Program documentation](https://github.com/todogroup/governance/blob/main/TODO-OSPO-Ambassador-Program.md)
and (2) fulfil the requirements from the Pre-submission checklist.

## Pre-submission checklist:

- [ X ] I am 18 years of age or older
- [ X ] I abide the [TODO Code Of Conduct](https://todogroup.org/code-of-conduct/)
- [ X ] I abide the [LF antitrust policy](https://www.linuxfoundation.org/legal/antitrust-policy)
- [ X ] I Confirm I meet AT LEAST ONE of the criteria documented at [TODO OSPO Ambassador Program documentation](https://github.com/todogroup/governance/blob/main/TODO-OSPO-Ambassador-Program.md)

## Contact Information

We would need the following information about you to get in contact if elected:

- Jose Rodriguez Roa
- Cuemby LLC
- jose.rodriguez@cuemby.com
- dagoshe
- (Optional) Frediverse Handle

## Current involvement in TODO Community

Who are you?

- [ X ] An active contributor to a TODO initiative (OSPOlogy, OSPO Landscape, Website, OSPO Book, etc)
- [ X ] A representative from a TODO General Member or OSPO Associate organization
- [ X ] An OSPO Local Organizer and/or OSPOlogy Organizer
- [ X ] An active leader in the TODO community with a minimum of 1 year of experience in:
    * Organizing events (virtual/in-person)
    * Speaking at events
    * Mentoring others 
    * Creating content (e.g., blogs, videos, etc.)

## REQUIRED: What do you expect from TODO Group as an TODO OSPO Ambassador? Whow would you help the community to thrive?

This is a very important section to understand how you would like to help the OSPO Ecosystem and engage in the TODO community. 
Here you can also provide links to resources (blogs on OSPOs, Community calls and/or OSPOlogy live micro-conferences where you participated, etc)

**Comment by dagoshe** [comment]
@anajsana correcting the email - Jose.rodriguez@cuemby.com

**Comment by anajsana** [comment]
Hello Jose! Thank you for your interest in becoming a TODO OSPO Ambassador. Our Steering Committee will review this request and will get back to you ASAP. 

One section that I noticed that is missing is the last question: What do you expect from TODO Group as a TODO OSPO Ambassador? How would you help the community to thrive?

Could you please complete this section before sending this to our SC? This is a very important section to understand how you would like to help the OSPO Ecosystem and engage in the TODO community.

Thank you!

**Comment by lhawthorn** [comment]
Thank you for your response, Jose. Would you please provide links or other information on your contributions to TODO Group?

**Comment by dagoshe** [comment]
Dear Leslie,

I'm excited to share that our journey in the program has just begun, and it's already shaping up to be a transformative experience, not just for us, but for the entire Open Source Community in Latin America. One of our standout contributions is the launch of the first in-person Kubernetes Community Days (KCD) in the region. This event marks a significant milestone in our commitment to nurturing a vibrant tech community (4 KCD more and 6 Ambassadores in the region https://www.youtube.com/watch?v=nA4acN-Xc0M). https://www.youtube.com/watch?v=wc7Qtba6pzw&list=PLj6h78yzYM2NGlhxZ1P0JXujDJWy-rMLU&index=31

A core focus of our initiative is to foster Diversity, Equity, and Inclusion (DEI) in the Latin American tech landscape. We're passionate about creating an inclusive environment where diverse voices are heard and valued. Our goal is to bring together over 1000 companies, forming a tech community that not only embraces DEI but also actively promotes the principles of Open Source. ( KCD El Salvador and this community https://www.cuemby.com/rise-up-fintech

We're also dedicated to facilitating knowledge sharing and engagement through live and online events each month. These events are designed to be interactive, informative, and inclusive, providing a platform for learning, collaboration, and innovation. https://lnkd.in/ga7gyrxz

On the technical front, in partnership with Cuemby, we're making significant strides in developing an open-source platform based on Kubernetes. This endeavor is not just about building technology; it's about creating a unique community project that resonates with our values of inclusion and accessibility. One of the exciting features we're working on is a module specifically designed for color-blind developers, ensuring that our platform is accessible and user-friendly for all.

As we move forward with these initiatives, we're enthusiastic about the potential impact and positive change we can bring to the tech community in Latin America. We believe that by working together, we can create a more inclusive, diverse, and innovative technological future. Please let me know how may I help with the ecosystem in the region and further more build up more and more community.

Thank you for your support and encouragement as we embark on this exciting journey.

Best regards

Jose Rodriguez

**Comment by anajsana** [comment]
@lhawthorn also to give some context and with regards to specific contributions to TODO / OSPOs, Cuemby recently became member and exploring ways to help contribute and increase participation. The most recent activity where Jose participated was as a [speaker at the first OSPOlogyLive NA Legal edition](https://youtu.be/niZ9mEFqJPU?si=ya4O6vSs5SrAj7ls) and attended the recent OSPOlogy webinars and engaged in conversations with the speakers.

My understanding is that @dagoshe (please correct me if I'm wrong) would like to start local meetups to activate OSPO adoption in LATAM regions as a way to contribute to the TODO and its community.

@dagoshe I do have one question about how would you advocate the message of why open source people/strategy/ community engagement and the set up of an OSPO (or similar open source initiative) is necessary at an organizational level. Are you planning to include the OSPO message / TODO best practices talks in the existing cloud native meet-ups? 🙂  Thanks!

**Comment by dagoshe** [comment]
Exactly Ana, thank you for clarifying, and to respond your answer is would be advocating for open source at the organizational level involves highlighting its role in driving innovation and collaboration. An Open Source Program Office (OSPO) is key in guiding these efforts, ensuring compliance and effective community engagement as of today no one is doing in the region. 
We're excited to integrate OSPO and TODO best practices into our cloud native meet-ups. These discussions will enrich our community's understanding of implementing and managing open source initiatives, fostering a culture of shared knowledge and growth, as I wrote on a previous email I im constant contact of companies in Latam or that have an intention of landing in the region (Spain-Germany-Israel) in which some how I embed the message and the principles of the lInux Foundation.

Looking forward to exploring these vital topics in our upcoming sessions!

Best regards,
